<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JournalViewer2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QTabWidget" name="MainTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="dataTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="SourceGroup">
              <property name="title">
               <string>Source</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QPushButton" name="instrumentButton">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="text">
                  <string>&lt;INSTRUMENT&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cycleButton">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="text">
                  <string>&lt;CYCLE&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="ViewGroup">
              <property name="title">
               <string>View</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="FilterCaseSensitivityButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>22</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Aa</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="RunFilterEdit">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="placeholderText">
                    <string>Filter run data</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="ClearFilterButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>20</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="icons.qrc">
                     <normaloff>:/icons/cross.svg</normaloff>:/icons/cross.svg</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QToolButton" name="GroupRunsButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="toolTip">
                    <string>Group runs with identical titles together</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="icons.qrc">
                     <normaloff>:/icons/group.svg</normaloff>:/icons/group.svg</iconset>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="runDataTable">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetLocalSource"/>
    <addaction name="actionClearLocalSource"/>
    <addaction name="separator"/>
    <addaction name="actionMountPoint"/>
    <addaction name="actionClearMountPoint"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QMenu" name="menuSearch_across_cycles">
     <property name="title">
      <string>Search across cycles</string>
     </property>
     <addaction name="actionMassSearchRB_No"/>
     <addaction name="actionMassSearchTitle"/>
     <addaction name="actionMassSearchUser"/>
     <addaction name="actionMassSearchRunRange"/>
     <addaction name="actionMassSearchDateRange"/>
    </widget>
    <widget class="QMenu" name="menuGo_to_specific_run_number">
     <property name="title">
      <string>Go to specific value</string>
     </property>
     <addaction name="actionRun_Number"/>
    </widget>
    <addaction name="actionSearch"/>
    <addaction name="actionSelectNext"/>
    <addaction name="actionSelectPrevious"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="menuSearch_across_cycles"/>
    <addaction name="actionClear_cached_searches"/>
    <addaction name="separator"/>
    <addaction name="menuGo_to_specific_run_number"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFind"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMassSearchRB_No">
   <property name="text">
    <string>RB No.</string>
   </property>
  </action>
  <action name="actionMassSearchTitle">
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionMassSearchUser">
   <property name="text">
    <string>User</string>
   </property>
  </action>
  <action name="actionMassSearchRunRange">
   <property name="text">
    <string>Run Range</string>
   </property>
  </action>
  <action name="actionMassSearchDateRange">
   <property name="text">
    <string>Date Range</string>
   </property>
  </action>
  <action name="actionClear_cached_searches">
   <property name="text">
    <string>Clear cached searches</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSelectNext">
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSelectPrevious">
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All Occurrences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F3</string>
   </property>
  </action>
  <action name="actionRun_Number">
   <property name="text">
    <string>Run Number</string>
   </property>
  </action>
  <action name="actionClearLocalSource">
   <property name="text">
    <string>Clear local journal source</string>
   </property>
  </action>
  <action name="actionSetLocalSource">
   <property name="text">
    <string>Set local journal source</string>
   </property>
  </action>
  <action name="actionMountPoint">
   <property name="text">
    <string>Set nexus archive location</string>
   </property>
  </action>
  <action name="actionClearMountPoint">
   <property name="text">
    <string>Clear nexus archive location</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
